packages/dictionary/src/metaSchema/dictionarySchemas.ts

```diff
@@ -110,37 +188,37 @@ export const SchemaFieldBase = zod
 	.strict();
 export type SchemaFieldBase = zod.infer<typeof SchemaFieldBase>;
 
-export const SchemaStringField = SchemaFieldBase.merge(
+export const SchemaBooleanField = SchemaFieldBase.merge(
 	zod.object({
-		valueType: zod.literal(SchemaFieldValueType.Values.string),
-		restrictions: StringFieldRestrictions.or(StringFieldRestrictions.array()).optional(),
+		valueType: zod.literal(SchemaFieldValueType.Values.boolean),
+		restrictions: BooleanFieldRestrictionsObject.or(BooleanFieldRestrictionsObject.array()).optional(),
 	}),
 ).strict();
-export type SchemaStringField = zod.infer<typeof SchemaStringField>;
+export type SchemaBooleanField = zod.infer<typeof SchemaBooleanField>;
 
 export const SchemaNumberField = SchemaFieldBase.merge(
 	zod.object({
 		valueType: zod.literal(SchemaFieldValueType.Values.number),
-		restrictions: NumberFieldRestrictions.or(NumberFieldRestrictions.array()).optional(),
+		restrictions: NumberFieldRestrictionsObject.or(NumberFieldRestrictionsObject.array()).optional(),
 	}),
 ).strict();
 export type SchemaNumberField = zod.infer<typeof SchemaNumberField>;
 
 export const SchemaIntegerField = SchemaFieldBase.merge(
 	zod.object({
 		valueType: zod.literal(SchemaFieldValueType.Values.integer),
-		restrictions: IntegerFieldRestrictions.or(IntegerFieldRestrictions.array()).optional(),
+		restrictions: IntegerFieldRestrictionsObject.or(IntegerFieldRestrictionsObject.array()).optional(),
```

Comment:
`IntegerFieldRestrictionsObject` could be the `IntegerFieldRestrictions` or `IntegerFieldConditionalRestriction` which is the new conditional restriction object we just defined.